src/app/detail/ui/styledComponent.tsx

```diff
@@ -81,4 +81,27 @@ export const TapWriteComment = styled(Tapwrite)(({ theme }) => ({
 
 export const TapWriteTaskEditor = styled(Tapwrite)(({ theme }) => ({
   minHeight: '30vh',
+  fontSize: '20px',
 }))
+
+export const TapWriteTaskEditor1 = styled(Tapwrite)(({ theme }) => ({
+  minHeight: '20vh',
+  borderColor: theme.color.borders.border,
+  border: `1px solid ${theme.color.base.black}`,
+}))
+
+export const StyledContainer = styled('div')<{ padding?: string; borderColor?: string; borderLess?: boolean }>(
```

Comment:
Using a prop like `borderLess` feels unintuitive? Is it possible to switch it out to something like `bordered` or `borderWidth` that takes 0 value by default?